### ISSUE TITLE
weekplans_bloc_test.dart should be refactored

### ISSUE THREAD

**Issue by Determinatoro** [aau-giraf/weekplanner-OLDUI#565]
**Describe the bug**
Some of the unit tests in this file use datetime now. This makes them use different values for each time, which is bad practice. It could be that the weekplans_bloc also has to be refactored to better support the unit tests.

**Comment by clahanse** [comment]
This issue was investigated and could not be reproduced. The weekplans_bloc_test.dart did not contain any instances of DateTime.now() but only regular calls to the DateTime method. We investigated earlier versions of the test-file and did not uncover any uses of DateTime.now() either. We are thus unable to determine whether the issue originally was a mistake or some changes have been made without being properly documented. While inspecting the test-file we came across two tests that have been commented out and we also found it hard to follow the test principles being employed in the file. For that reason we opened a new issue that serves to examine and refactor weekplans_bloc_test.dart in more general.
